packages/legend-extension-dsl-serializer/src/models/metamodels/pure/model/packageableElements/store/Binding.ts

```diff
@@ -17,15 +17,29 @@
 import { observable, action, makeObservable, override } from 'mobx';
 import { hashArray } from '@finos/legend-shared';
 import type { Hashable } from '@finos/legend-shared';
-import type { PackageableElementVisitor } from '@finos/legend-graph';
-import { Store } from '@finos/legend-graph';
+import type {
+  OptionalPackageableElementReference,
+  PackageableElementVisitor,
+} from '@finos/legend-graph';
+import {
+  OptionalPackageableElementExplicitReference,
+  Store,
+} from '@finos/legend-graph';
 import type { ModelUnit } from './ModelUnit';
 import { DSL_SERIALIZER_HASH_STRUCTURE } from '../../../../../DSLSerializer_ModelUtils';
+import { SchemaSet } from '../schemaSet/SchemaSet';
+
+export enum BINDING_CONTENT_TYPE {
+  FLAT_DATA = 'application/x.flatdata',
+  XSD = 'application/xml',
+  JSON = 'application/json',
+}
 
 export class Binding extends Store implements Hashable {
-  schemaSet?: string | undefined;
+  schemaSet: OptionalPackageableElementReference<SchemaSet> =
```

Comment:
Nope, we try to move away from this remember :), let's do `schemaSet!: OptionalPackageableElementReference<SchemaSet>`